src/importer/dnd5ImportTypes.d.ts

```diff
@@ -69,18 +86,3 @@ export type spellImport = {
 	material: string; // materials
 	level: 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9; 
 }
-
-export type spellData = {
```

Comment:
ich war blind, nimm den ma wieder rein. Wenn ich Unsinn laber in nem Review dann kannst mir das sagn xD